weblab/static/js/main.js

```diff
@@ -10,6 +10,7 @@ var experiment = require('./experiment.js');
 var notifications = require('./lib/notifications.js');
 require('./compare.js');
 require('./entity_version_list.js');
+require('./experiment_tasks');
```

Comment:
I note all the others have `.js` appended - I'm presuming that this therefore isn't actually required?

Reply:
yes, good spot thanks. I have corrected the extension. Not sure why it was working locally without that - does it perhaps use the local .js files? rather than the gulped one?